INFO     requests.session:session.py:59 2025-06-11 13:38:41.393发送请求>>>>>>>       接口地址=GET http://39.99.136.49:9000/dashboard/getStatData
INFO     requests.session:session.py:60 2025-06-11 13:38:41.393发送请求>>>>>>>       请求头={'Authorization': 'None', 'x-sign': '417B110F1E71BD2CFE96366E67849B0B'}
INFO     requests.session:session.py:61 2025-06-11 13:38:41.393发送请求 >>>>>>>      请求体=无
INFO     requests.session:session.py:90 2025-06-11 13:38:41.442接收响应<<<<<<<       状态码=200
INFO     requests.session:session.py:91 2025-06-11 13:38:41.442接收响应<<<<<<<       响应头={'Access-Control-Allow-Credentials': 'true', 'X-Content-Type-Options': 'nosniff', 'X-XSS-Protection': '0', 'Cache-Control': 'no-cache, no-store, max-age=0, must-revalidate', 'Pragma': 'no-cache', 'Expires': '0', 'Content-Type': 'application/json;charset=utf-8', 'Content-Length': '60', 'Date': 'Wed, 11 Jun 2025 05:38:39 GMT'}
INFO     requests.session:session.py:93 2025-06-11 13:38:41.442接收响应<<<<<<<       响应体={"code":401,"msg":"登录无效，不能访问","data":null}...
INFO     root:test_api.py:53 断言：预期：success 实际：None